frontend/src/scalars/ui/Config.vue

```diff
@@ -1,26 +1,26 @@
 <template>
     <div class="visual-dl-page-config-com">
         <v-text-field
-                name="Group name RegExp"
                 label="Group name RegExp"
                 hint="input a tag group name"
-                id="testing"
                 v-model="config.groupNameReg"
                 dark
         ></v-text-field>
 
-        <v-slider label="Smoothing" v-bind:max="100" v-bind:min="0" v-model="config.smoothing" dark></v-slider>
-        <v-text-field v-model="config.smoothing" type="number" dark></v-text-field>
+        <div>
+            <v-slider label="Smoothing" :max="0.999" :min="0" :step="0.001" v-model="config.smoothing" dark></v-slider>
```

Comment:
Does v-slider take step smaller than 1? I thought it only take Integer number

Reply:
v-slider takes type ```Number``` which can be double or integer